lib/logstash/outputs/elasticsearch.rb

```diff
@@ -367,36 +367,37 @@ def multi_receive(events)
     wait_for_successful_connection if @after_successful_connection_done
     events_mapped = safe_interpolation_map_events(events)
     retrying_submit(events_mapped.successful_events)
-    unless events_mapped.failed_events.empty?
-      @logger.error("Can't map some events, needs to be handled by DLQ #{events_mapped.failed_events}")
-      send_failed_resolutions_to_dlq(events_mapped.failed_events)
+    unless events_mapped.event_mapping_errors.empty?
+      handle_event_mapping_errors(events_mapped.event_mapping_errors)
```

Comment:
If dlq is enabled then no log lines are logged. I think that  any mapping error, that route events to DLQ, should be clearly highlighted into the logs, so a `warn` log line should be present to eventually understand what's happening.